locale/en/download/package-manager.md

```diff
@@ -358,7 +358,7 @@ Using **[MacPorts](http://www.macports.org/)**:
 port install nodejs<major version>
 
 # Example
-port install nodejs7
+port install nodejs8
```

Comment:
I think so unless you or someone who uses macport can confirm that the port is available.